tests/e2e/holodeck.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: holodeck.nvidia.com/v1alpha1
+kind: Environment
+metadata:
+  name: HOLODECK_NAME
+  description: "Devel infra environment"
+spec:
+  provider: aws
+  auth:
+    keyName: cnt-ci
+    privateKey: HOLODECK_PRIVATE_KEY
+  instance:
+    type: g4dn.xlarge
+    region: us-west-1
+    ingressIpRanges:
+      - 0.0.0.0/0
+    image:
+      architecture: amd64
+      imageId: ami-0ce2cb35386fc22e9
```

Comment:
Out of scope for this PR: This determines the baseOS, correct? Should we rather maintain a map of OS to AMI to make the target here clearer?

Reply:
sure, that sounds like a TODO for https://github.com/NVIDIA/holodeck/tree/main

Reply:
https://github.com/NVIDIA/holodeck/issues/32